specs/updated_cluster_list_implementation_with_upadated_API_response

```diff
@@ -0,0 +1,140 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of unmanaged gluster clusters discovered by Tendrl
+
+Display list of unmanaged gluster clusters present in Tendrl. The list should have functionality for 
+
+* filtering by name.
+* sorting by name, status, cluster version and managed.
+
+Cluster list will be sorted by name by default.
+
+View design will look like:
+
+image::upadtedClusterList.png[Upadted Cluster List]
+
+== Problem description
+
+Tendrl should show list of unmanaged gluster clusters present in Tendrl along with their details.
+
+The details shown for each cluster in UI is -
+
+* Cluster Name
+* Status
+* Managed State
+* Cluster Version
+* Import button
+* Message to state cluster configurable state for eg. ready to import, cluster misconfigured.
```

Comment:
@r0h4n @anivargi Which API will provide the error messages? Can you please provide the sample response of the same?